src/App.tsx

```diff
@@ -0,0 +1,59 @@
+import { SearchBar } from "./SearchBar";
+import ResultsTable from "./ResultsTable";
+import data from "./data.json";
+import { useState } from "react";
+import Tabs from "./Tabs";
+
+export default function App() {
+  const [parkingSpots, setParkingSpots] = useState(getTenRandomParkingSpots());
+  const [selectedTab, setSelectedTab] = useState("Nearby");
+  const tabOptions = ["Nearby", "History", "Favourites"];
+
+  function submitSearch(location: string): void {
+    setParkingSpots(getTenRandomParkingSpots());
+  }
+
+  function getTenRandomParkingSpots(): ParkingData[] {
+    const shuffled = [...data.parkingSpots].sort(() => 0.5 - Math.random());
+    return shuffled.slice(0, 10);
+  }
+
+  return (
+    <div className="mx-auto w-4/5">
```

Comment:
You probably should create a css class for the ParkingSlot, and for the ParkingLot, and use them here instead of these values. Inside the CSS class, you could use the values.